CHANGELOG.md

```diff
@@ -4,6 +4,59 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+### Changed
+
+* On AWS and OpenStack, ports 10251 (controller manager) and 10252
+  (scheduler) have been opened to access from all machines.  This
+  allows Prometheus (which runs on the worker nodes) to scrape all
+  machines for metrics.
+
+## 0.6.0 - 2018-12-09
+
+### Added
+
+- We now push a `kubeadmin` user (with an internally-generated
+  password) into the cluster for the new [bootstrap identity
+  provider][bootstrap-identity-provider].  This gives users a way to
+  access the web console, Prometheus, etc. without needing to
+  configure a full-fledged identity provider or install `oc`.  The
+  `create cluster` subcommand now blocks until the web-console route
+  is available and then exits after printing instructions for using
+  the new credentials.
+- The installer binary now includes Terraform so there is no longer a
+  need to seperately download and install it.
```

Comment:
`/s/seperately/separately`

Reply:
>     /s/seperately/separately

Fixed with 414b8c8 -> 05ba0fa, which also cleans up the section about #791.